src/components/modals/WalletListModal.tsx

```diff
@@ -64,6 +64,7 @@ interface Props {
   headerTitle: string
   showBankOptions?: boolean
   showCreateWallet?: boolean
+  parentWallet?: EdgeCurrencyWallet
```

Comment:
You don't actually use the full wallet. Just pass the `wallet.id` to ensure you don't access a wallet object that may have been deleted.